projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
Internal links have been moved to https://github.com/hellofresh/awesome-go, so if you think the rest of this PR is fine, it's time to approve this PR.